### ISSUE TITLE
Apply accessibility specs to focused components

### ISSUE THREAD

**Issue by ishaan000** [hiyaryan/the-cdj-ui#6]
We can create themes that follow accessibility specs and apply them throughout the app. 

This is what chatgpt says, need to verify :

"&:focus: Adds a CSS rule to apply a 2px solid black outline when the button is focused, improving accessibility and making it clearer when the element is active."

`{
  margin: 2,
  '&:focus': {
    outline: '2px solid #000',
  }
}
`
